wire-ios-data-model/Tests/MLS/MLSServiceTests.swift

```diff
@@ -2650,4 +2650,143 @@ class MLSServiceTests: ZMConversationTestsBase, MLSServiceDelegate {
         XCTAssertTrue(waitForCustomExpectations(withTimeout: 0.5))
     }
 
+    // MARK: - Proteus to MLS migration
+
+    func test_startProteusToMLSMigration_succeeds() async throws {
+        // Given
+        BackendInfo.domain = "example.com"
```

Comment:
```suggestion
        BackendInfo.storage = .random()!
        BackendInfo.domain = "example.com"
```

Reply:
Done: ab7419b4df194acaf9056cf0dbec0b2767fff22a

However, it should probably be cleaned up as well using `removePersistentDomain(forName:)`.